example/screens/MessageCenterScreen.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Sample React Native App
+ *
+ * MessageCenterScreen: Contains the list of messages.
+ */
+'use strict';
+
+import {
+ UrbanAirship,
+} from 'urbanairship-react-native'
+
+import React, {
+  Component,
+} from 'react';
+
+import {
+  Text,
+  View,
+  Image,
+  ScrollView,
+  Button,
+  FlatList,
+  TouchableHighlight,
+} from 'react-native';
+
+import Moment from 'moment';
+
+import styles from './../Styles';
+import MessageScreen from "./MessageScreen"
```

Comment:
Is this needed? I don't see it used. I'm not a JS expert, so I could be missing it.

Reply:
Yes you're right, we don't need the `import MessageScreen from "./MessageScreen"` I removed it